internal/go/skycfg/proto_message.go

```diff
@@ -538,7 +538,21 @@ func typeName(t reflect.Type) string {
 }
 
 func typeError(t reflect.Type, sky starlark.Value) error {
-	return fmt.Errorf("TypeError: value %s (type `%s') can't be assigned to type `%s'.", sky.String(), sky.Type(), typeName(t))
+	if sky.Type() == typeName(t) {
```

Comment:
Go style nit: this would be a bit easier to read if it's unindented, and if the multi-line error used a raw string literal.

```go
func typeError(t reflect.Type, sky starlark.Value) error {
	if sky.Type() != typeName(t) {
		return fmt.Errorf("TypeError: value %s (type '%s') can't be assigned to type '%s'.", sky.String(), sky.Type(), typeName(t))
	}
	return fmt.Errorf(`TypeError: value %s (type '%s') can't be assigned to type '%s'.
etc`,
	sky.String(), sky.Type(), typeName(t))
}
```

Reply:
Sure, will do.

Reply:
I want the top of the error to match. The error has backticks, which get ugly with raw strings, so my (maybe mistaken) feeling is to go with string concatenation again.

Reply:
That's fine. The shorter message kept it readable.